styles.css

```diff
@@ -8,8 +8,42 @@
   src: url(fonts/PlayfairDisplay-Regular.ttf);
 }
 
+@font-face {
+  font-family: "PlayfairDisplay-Bold";
+  src: url(fonts/PlayfairDisplay-Bold.ttf);
+}
+
+@font-face {
+  font-family: "Lato-Light";
+  src: url(fonts/Lato-Light.ttf);
+}
+
+@font-face {
+  font-family: "Proxima-Nova-Extrabold";
+  src: url(fonts/Proxima-Nova-Extrabold.ttf);
+}
+
+@font-face {
+  font-family: "Proxima-Nova-Semibold";
+  src: url(fonts/Proxima-Nova-Semibold.ttf);
+}
+
+@font-face {
+  font-family: "Proxima-Nova-Regular";
+  src: url(fonts/Proxima-Nova-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Lato-Regular";
+  src: url(fonts/Lato-Regular.ttf);
+}
+
 body {
+  display: flex;
+  flex-direction: column;
```

Comment:
I don't see why you need flex on body

Reply:
4 Using flexbox https://sgeek.pro/web/coding/5-sposobov-kak-prizhat-footer-k-nizu-stranitsy/

Reply:
1) there wasn't task to make footer at bottom
2) it doesn't work